TSPL.docc/LanguageGuide/Concurrency.md

```diff
@@ -271,6 +276,11 @@ func generateSlideshow(forGallery gallery: String) async {
 }
 ```
 
+<!--
+XXX TR: Do we need to mention this?
+How common is it for developers to need to insert manual yields?
```

Comment:
Inserting yield() manually is definitely not the first thing people need to encounter as they read up about concurrency, so I'd agree that we can move it out from here. 

I'd stuff it all the way at the end of the page maybe in some "forcing suspensions" section which could be something about debugging performance and "overhangs" with swift concurrency perhaps.